src/app/views/rolodex_view.js

```diff
@@ -1,6 +1,107 @@
 import Backbone from 'backbone';
+import _ from 'underscore';
+import ContactView from 'app/views/contact_view';
+import Contact from 'app/models/contact';
+import $ from 'jquery';
 
 const RolodexView = Backbone.View.extend({
+  initialize: function(){
+    //Sets the template to pass to the ContactView
+    this.template = _.template($('#tmpl-contact-card').html());
+
+    //Sets the element that all the contact cards are put into
+    this.element = this.$('#contact-cards');
+
+    //Sets the template & element for the modal
+    this.detailsTemplate = _.template($('#tmpl-contact-details').html());
+    this.detailsElement = $('#contact-details');
+
+    //Gets the inputs from the contact form
+    this.input = {
+      name: this.$('.contact-form input[name="name"]'),
+      phone: this.$('.contact-form input[name="phone"]'),
+      email: this.$('.contact-form input[name="email"]')
+    };
```

Comment:
For this type of behavior in the future use `this.$` so that you're restricted to searching 'inside' the view.